react-app/src/styles/mixins.scss

```diff
@@ -0,0 +1,139 @@
+@import 'vars.scss';
```

Comment:
Same with this file, not quite sure what it does now but we should edit it once we have the styles we want

Reply:
These are some useful mixins that we can use for styling